website/source/docs/platform/k8s/helm/examples/index.html.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: "docs"
+page_title: "Examples"
+sidebar_current: "docs-platform-k8s-examples"
+sidebar_title: "Examples"
+description: |-
+  This section documents configuration options for the Vault Helm chart
+---
+
+# Helm Chart Examples
+
+~> **Important Note:** This chart is not compatible with Helm 3. Please use Helm 2 with this chart.
+
+These are a collection of examples of common configurations for Vault using the Helm chart. 
+
+The following are different configuration examples to support a variety of
+deployment models. You can view the different examples from the list on the left.
```

Comment:
In the Netlify preview, it looks a little weird to say "the following" because of how the page is white immediately below. Do you think we should just strip the "Overview" page altogether? Because from a UX standpoint it still works without it, at least in the Netlify preview. If this page is valuable because of the **Important Note**, I wonder if we should move it to elsewhere.

Reply:
Actually, I see the note is at the top of every page, which is great, so maybe it really would be okay to strip this page altogether?

Reply:
Unfortunately we can't get rid of it. All sections need to have an Overview page :(